internal/pkg/deploy/cloudformation/env.go

```diff
@@ -1,127 +1,50 @@
+// Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+// Package cloudformation provides functionality to deploy archer resources with AWS CloudFormation.
```

Comment:
Can we get rid of this? Don't need no mo'

Reply:
need which part?

Reply:
NEVERMIND